apps/site/assets/ts/schedule/components/line-diagram/ExpandableBranch.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { ReactElement } from "react";
+import { times } from "lodash";
+import ExpandableBlock from "../../../components/ExpandableBlock";
+import { LineDiagramStop, RouteStop } from "../__schedule";
+import SingleStop from "./SingleStop";
+
+const ExpandableBranch = ({
+  branchData,
+  onStopClick,
+  color,
+  willMerge
+}: {
+  branchData: LineDiagramStop[];
+  onStopClick: (stop: RouteStop) => () => void;
+  color: string;
+  willMerge?: boolean;
+}): ReactElement<HTMLElement> | null => (
+  <div
+    className={`m-schedule-diagram__expander ${
+      willMerge ? "m-schedule-diagram__expander--merging" : ""
+    }`}
+  >
+    <ExpandableBlock
+      header={{
+        iconSvgText: null,
+        text: (
+          <div
+            className={`m-schedule-diagram__stop ${willMerge &&
+              "m-schedule-diagram__stop--merging"}`}
+          >
+            <div
+              className="m-schedule-diagram__lines m-schedule-diagram__lines--collapsed"
+              style={{ color: `#${color}` }}
+            >
+              <>
+                {times(branchData[0].stop_data.length - 1, idx => (
+                  <div key={idx} className="m-schedule-diagram__line" />
+                ))}
+                <div className="m-schedule-diagram__line m-schedule-diagram__line--collapsed">
+                  <div className="m-schedule-diagram__collapsed-icon">
+                    <svg
+                      viewBox="0 0 10 30"
+                      height="30"
+                      className="m-schedule-diagram__line-stop"
+                    >
+                      <circle r="4" cy="10" cx="50%" />
+                      <circle r="4" cy="20" cx="50%" />
+                      <circle r="4" cy="30" cx="50%" />
+                    </svg>
+                  </div>
+                </div>
+              </>
+            </div>
+            <div className="m-schedule-diagram__content">
+              <button
+                className="btn btn-link m-schedule-diagram__toggle"
+                type="button"
+              >
+                {branchData.length} {branchData.length > 1 ? "stops" : "stop"}
+              </button>
+            </div>
+          </div>
+        )
+      }}
+      initiallyExpanded={false}
+      id={`${branchData[0].route_stop.branch}`}
+    >
+      <>
+        {branchData.map((branchStop, idx) => (
+          <SingleStop
+            key={branchStop.route_stop.id}
+            stop={branchStop}
+            isLast={idx === branchData.length - 1}
+            onClick={onStopClick(branchStop.route_stop)}
+            color={color}
+          />
+        ))}
+      </>
```

Comment:
This was because the [ExpandableBlock children prop](https://github.com/mbta/dotcom/blob/master/apps/site/assets/ts/components/ExpandableBlock.tsx#L22) is typed to only expect one ReactElement rather than a list of 'em! 🤷🏻‍♀️

Reply:
Ah, it seems this is a built-in React thing and so we can't control the name (or, I guess, whether it's able to accept a list of components).